Meta/lint-ci.sh

```diff
@@ -25,6 +25,7 @@ for cmd in \
         Meta/check-markdown.sh \
         Meta/check-newlines-at-eof.py \
         Meta/check-style.py \
+        Meta/check-png-sizes.sh \
```

Comment:
o r d e r

Reply:
Y E S

Reply:
just to be clear, it's not actually ordered - P before S